app/services/analytics_events.rb

```diff
@@ -2350,5 +2350,17 @@ def idv_in_person_usps_proofing_results_job_enrollment_failure(reason:, enrollme
       **extra,
     )
   end
+
+  # Tracks individual enrollments that succeed during GetUspsProofingResultsJob
+  # @param [String] reason why did this enrollment pass?
+  # @param [String] enrollment_id
+  def idv_in_person_usps_proofing_results_job_enrollment_success(reason:, enrollment_id:, **extra)
```

Comment:
sorry to keep harping on this... I think we should combine `_success` and `_failure` into one event with a `success:` boolean to tell them apart so they'll be easier to aggregate and find in the future

Reply:
ugh, sorry, clicked squash & merge just as I saw your comment. I will follow up with a new PR.

Reply:
this was fixed in #6832